File: dhHelpdeskNG/dhHelpdesk_NG.DTO/Models/MailTemplates/MailFile.cs
namespace DH.Helpdesk.BusinessData.Models.MailTemplates
{
    public class MailFile
    {
        public string FileName { get; set; }
        public string FilePath { get; set; }
        public bool IsInternal { get; set; }

        public static MailFile CreateInternal(string fileName, string filePath)
        {
            return new MailFile()
            {
                FileName = fileName,
                FilePath = filePath,
                IsInternal = true
            };
        }
    }
}

File: dhHelpdeskNG/DH.Helpdesk.Library/Mail/MailFile.vb
Public Class MailFile 

    Public Sub New(sFileName As String, sFilePath As String, Optional bIsInternal As Boolean = False)
        FileName = sFileName
        FilePath = sFilePath
        IsInternal = bIsInternal
    End Sub

    Public Property FileName as String
    Public Property FilePath as String
    Public Property IsInternal as String
End Class